### PR TITLE
feat(cfnspec): add CloudFormation documentation to L1 classes

### PR DESCRIPTION
**Pull request by rix0rrr**
Include CloudFormation documentation in the generated L1 docstrings.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by rix0rrr** [comment]
Pre-emptive FAQ:

*Q: How are we going to keep the data file up-to-date?*

A: I would like to consider that of scope for this PR. Rest assured I have a cunning plan 🤓

File: tools/@aws-cdk/cfn2ts/lib/codegen.ts [hunk 1]
```diff
@@ -144,15 +144,17 @@ export default class CodeGenerator {
     container: Container): Dictionary<string> {
     const propertyMap: Dictionary<string> = {};
 
+    const docs = typeDocs(resource.specName?.fqn ?? '');
+
     Object.keys(propertiesSpec).sort(propertyComparator).forEach(propName => {
       this.code.line();
       const propSpec = propertiesSpec[propName];
-      const additionalDocs = resource.specName!.relativeName(propName).fqn;
+      const additionalDocs = docs.properties[propName] || quoteCode(resource.specName!.relativeName(propName).fqn);
```
Comment (iliapolo) [inline tools/@aws-cdk/cfn2ts/lib/codegen.ts:152]: You sure `docs.properties[propName]` doesn't require `quoteCode`? Can it hurt?

**Review by iliapolo — COMMENTED** [review]
@rix0rrr This is very cool, but...in-spite of your pre-emptive clarification, I am somewhat uncomfortable pushing this without a maintenance mechanism. 

Once the docs get stale in significant way, this will result in a degraded experience than what we have now. And we have no way of knowing when this will happen. I think we should hold back until we at least have a manual process or something.

Also, can we push a few tests so we can see a diff when this code changes?

**Comment by rix0rrr** [comment]
Here's an example of a generated L1 file: https://pastebin.com/fJRmT3E4

File: tools/@aws-cdk/cfn2ts/lib/codegen.ts (hunk 1 above)
Comment (rix0rrr) [reply tools/@aws-cdk/cfn2ts/lib/codegen.ts:152]: Given that `quoteCode` looks like this:

```ts
function quoteCode(code: string): string {
  return '`' + code + '`';
}
```

I would say, no. This is intentional, we don't want to monospace the other docs.

**Comment by iliapolo** [comment]
@rix0rrr as we discussed lets currently hold off on this until we have the update mechanism in-place as well. Converting to draft in the meanwhile.

**Comment by rix0rrr** [comment]
There is an update mechanism in place now

**Review by iliapolo — APPROVED** [review]

**Pull request merged**